### PR TITLE
Make it open-source-able

### PR DESCRIPTION
**Pull request by elbrujohalcon**
- [x] Change all internal references to generic ones.
- [x] Write a proper README.md

### CODE REVIEW

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'katana' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'shotgun' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'shotgun' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'xref_runner' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'xref_runner' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'katana' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ TEST_DEPS = xref_runner
 
 dep_katana = git https://github.com/inaka/erlang-katana.git 0.2.5
 dep_eper = git https://github.com/massemanet/eper.git 0.90.0
-dep_shotgun = git https://github.com/inaka/shotgun.git 0.1.8
+dep_shotgun = git https://github.com/inaka/shotgun.git 0.1.11
 dep_sync = git https://github.com/inaka/sync.git 0.1
 dep_xref_runner = git https://github.com/inaka/xref_runner.git 0.2.2
```
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'xref_runner' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'katana' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'shotgun' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'katana' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'xref_runner' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'shotgun' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

File: Makefile (hunk 1 above)
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+
+## Configuration
+The different game stages are configured by application environment. To help you with that a [sys.config.template](config/sys.config.template) is provided, along with the configuration we used at [inaka](config/inaka.config) and the [generic](config/generic.config) one that's used by default.
+If you want to start the server using your own config, just do:
+
+```bash
+CONFIG="path/to/your.config" make shell
+```
+
+You can also set the IP part of your node name by doing:
```
Comment (igaray) [inline README.md:27]: port

Comment (elbrujohalcon) [inline README.md:27]: I meant the ip portion of the node name

Comment (igaray) [inline README.md:27]: ha, got it

**Comment by gadgetci** [comment]
According to **Elvis**:

> Dependency 'sync' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:

> Dependency 'eper' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:

> Dependency 'xref_runner' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:

> Dependency 'shotgun' is not using git protocol, please change this to something like `git://...`

**Comment by gadgetci** [comment]
According to **Elvis**:

> Dependency 'katana' is not using git protocol, please change this to something like `git://...`

File: Makefile (hunk 1 above)
Comment (gadgetci) [inline Makefile:17]: According to **Compiler**:

> warning: overriding commands for target `shell'

**Pull request merged**